incubator/java-openliberty/templates/kafka/README.md

```diff
@@ -0,0 +1,78 @@
+# Kafka template for Open Liberty
+
+This template can be used to develop Liberty applications that connect to Kafka by using MicroProfile Reactive messaging. A simple `StarterApplication` is included that enables basic production and consumption of events. 
+
+
+## Getting Started with the StarterApplication.
+
+### 1. Create a new folder and initialize it using appsody init:
+
+
+```
+mkdir test-appsody-reactive
+cd test-appsody-reactive
+appsody init java-openliberty reactive
+```
+
+### 2. Start Kafka and Zookeeper
```

Comment:
`ZooKeeper` (and elsewhere in the PR)

Reply:
Also, maybe explain that ZooKeeper is just a dependency of Kafka and not actually a component in its own right.